src/Workleap.OpenApi.MSBuild/OasdiffManager.cs

```diff
@@ -62,7 +59,7 @@ private async Task DecompressDownloadedFileAsync(string oasdiffFileName, Cancell
             return;
         }
         
-        var exitCode = await this._processWrapper.RunProcessAsync("tar", new[] { "-xzf", $"{pathToCompressedFile}", "-C", $"{this._oasdiffDirectory}" }, cancellationToken);
+        var exitCode = await this._processWrapper.RunProcessAsync("tar", new[] { "-xzf", $"{pathToCompressedFile}", "-C", $"{this._oasdiffDirectory}" }, cancellationToken: cancellationToken);
```

Comment:
Why the change of specifying the parameter name?

Reply:
I like to name `CancellationToken` param. Makes it easier to extend in future if we add a default parameter.